samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/util/TotalScoresExportBean.java

```diff
@@ -0,0 +1,186 @@
+/**********************************************************************************
+ * $URL$
+ * $Id$
+ ***********************************************************************************
+ *
+ * Copyright (c) 2023 The Sakai Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **********************************************************************************/
+
+package org.sakaiproject.tool.assessment.ui.bean.util;
+
+import java.io.Serializable;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Iterator;
+
+import javax.faces.bean.ManagedBean;
+import javax.faces.bean.SessionScoped;
+import javax.faces.context.FacesContext;
+import javax.servlet.http.HttpServletResponse;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.poi.hssf.usermodel.HSSFCell;
+import org.apache.poi.hssf.usermodel.HSSFCellStyle;
+import org.apache.poi.hssf.usermodel.HSSFFont;
+import org.apache.poi.hssf.usermodel.HSSFRow;
+import org.apache.poi.hssf.usermodel.HSSFSheet;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+
+import org.sakaiproject.tool.assessment.ui.bean.evaluation.AgentResults;
+import org.sakaiproject.tool.assessment.ui.listener.util.ContextUtil;
+
+@Slf4j
+@ManagedBean(name="totalScoresExportBean")
+@SessionScoped
+public class TotalScoresExportBean implements Serializable {
+
+	/**
+	 * Creates a new TotalScoresExportBean object.
+	 */
+	public TotalScoresExportBean() {
+		log.debug("Creating a new TotalScoresExportBean");
+	}
+
+	public void exportExcel(String assessmentName, List allAgents) {
+		// Now insert the header line
+		List<String> headerList = new ArrayList<>();
+		headerList.add(ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EvaluationMessages","first_name"));
```

Comment:
Please use a constant for the bundle string :)

Reply:
Thanks see the new commit:
https://github.com/sakaiproject/sakai/pull/12108/commits/7dc85ec82c500e0c1d785c3e4ec03555d7872424